docs/source/international_languages.md

```diff
@@ -0,0 +1,232 @@
+   ```
+
+1. Remove all folders from the root directory except `/doc`. Likewise, remove
+   all files (including hidden ones) except `LICENCE` and `README.md`, so that
+   you are left with the following:
+
+   ```bash
+   ls -l
+   total 40
+   -rw-r--r--   1 user  staff  11358  5 Jun 14:38 LICENSE
+   -rw-r--r--   1 user  staff   4822  5 Jun 15:09 README.md
+   drwxr-xr-x  11 user  staff    352  5 Jun 14:38 docs
+   ```
+
+1. Update the `README.md` file to using [this
+   one](https://github.com/hyperledger-labs/fabric-docs-ml/blob/master/README.md)
+   as an example.
+
+   Customize the `README.md` file for your new language.
+
+1. Add a `.readthedocs.yml` file [like this
+   one](https://github.com/hyperledger-labs/fabric-docs-ml/blob/master/.readthedocs.yml)
+   to the top level folder.  This file is configured to disable ReadTheDocs PDF
+   builds which may fail if you use non-latin character sets. Your top level
+   repository folder will now look like this:
+
+   ```bash
+   (base) anthonyodowd/git/fabric-docs-sa ls -al
+   total 96
+   ...
+   -rw-r--r--   1 anthonyodowd  staff    574  5 Jun 15:49 .readthedocs.yml
+   -rw-r--r--   1 anthonyodowd  staff  11358  5 Jun 14:38 LICENSE
+   -rw-r--r--   1 anthonyodowd  staff   4822  5 Jun 15:09 README.md
+   drwxr-xr-x  11 anthonyodowd  staff    352  5 Jun 14:38 docs
+   ```
+
+1. Commit these changes locally to your local repository:
+
+   ```bash
+   git add .
+   git commit -s -m "Initial commit"
+   ```
+
+1. Create a new repository on your GitHub account called `fabric-docs-sa`. In
+   the description field type `Hyperledger Fabric documentation in Sanskrit
+   language`.
+
+1. Update your local git `origin` to point to this repository, replacing
+   `YOURGITHUBID` with your GitHub ID:
+
+   ```bash
+   git remote set-url origin git@github.com:YOURGITHUBID/fabric-docs-sa.git
+   ```
+
+1. Push your `release-2.1` branch to be the `master` branch in this repository:
+
+   ```bash
+   git push origin release-2.1:master
+
+   Enumerating objects: 6, done.
+   Counting objects: 100% (6/6), done.
+   Delta compression using up to 8 threads
+   Compressing objects: 100% (4/4), done.
+   Writing objects: 100% (4/4), 1.72 KiB | 1.72 MiB/s, done.
+   Total 4 (delta 1), reused 0 (delta 0)
+   remote: Resolving deltas: 100% (1/1), completed with 1 local object.
+   To github.com:ODOWDAIBM/fabric-docs-sa.git
+   b3b9389be..7d627aeb0  release-2.1 -> master
+   ```
+
+1. Verify that your new repository `fabric-docs-sa` is correctly
+   populated on GitHub under the `master` branch.
+
+1. Connect your repository to ReadTheDocs using these
+   [instructions](./docs_guide.html#building-on-github). Verify that your
+   documentation builds correctly.
+
+1. When you are happy with your repository, create a request to make an
+   equivalent repository in the Hyperledger Labs organization, following [these
+   instructions](https://github.com/hyperledger-labs/hyperledger-labs.github.io).
```

Comment:
We seem to be missing the step where they perform their translations. 

Should they complete their translations first before requesting the equivalent repo in Hyperledger Labs, or should they make that request first and then do all their translation work from the Hyperledger Labs repo?

Reply:
It's really up to them -- ML pushed withouy many, whereas Br-Pt are waiting. I could certainly add a step saying <do translations here>!

Reply:
I actually made a few concrete suggestions to add translations of front page and Introduction.  That seems like a basic minimum.  We also want them to get something in HL Labs as soon as possible.